### PR TITLE
PLFM-6000 How to add preuploaded gcloud files + fixes

### PR DESCRIPTION
**Pull request by nickgros**
https://sagebionetworks.jira.com/browse/PLFM-6000

### CODE REVIEW

File: articles/custom_storage_location.md [hunk 1]
```diff
@@ -350,93 +351,10 @@ project_destination = syn.restPOST('/projectSettings', body = json.dumps(project
 
 ##### R
 
-Please see the [REST docs](http://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/project/ExternalS3StorageLocationSetting.html) for more information on setting external storage location settings using our REST API.
```
Comment (nickgros) [inline articles/custom_storage_location.md]: This content was duplicated so it appeared twice in this document, hence the deletion

File: articles/custom_storage_location.md [hunk 2]
```diff
@@ -263,7 +264,7 @@ fileHandle <- list(concreteType='org.sagebionetworks.repo.model.file.S3FileHandl
                    storageLocationId = destination$storageLocationId,
                    bucketName = destination$bucket,
                    key ='s3ObjectKey')
-fileHandle <- synRestPOST('/externalFileHandle/s3', body=toJSON(fileHandle), endpoint = synapseFileServiceEndpoint())
+fileHandle <- synRestPOST('/externalFileHandle/s3', body=toJSON(fileHandle), endpoint = 'https://file-prod.prod.sagebase.org/file/v1')
```
Comment (nickgros) [inline articles/custom_storage_location.md]: synapseFileServiceEndpoint() doesn't work (at least for me) in synapser, so I removed references to it.

File: _articles/custom_storage_location.md [hunk 3]
```diff
@@ -350,93 +351,10 @@ project_destination = syn.restPOST('/projectSettings', body = json.dumps(project
 
 ##### R
 
-Please see the [REST docs](http://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/project/ExternalS3StorageLocationSetting.html) for more information on setting external storage location settings using our REST API.
```
Comment (nickgros) [inline _articles/custom_storage_location.md:353]: Duplicated content: see line 274

File: articles/custom_storage_location.md (hunk 2 above)
Comment (kelshmo) [reply articles/custom_storage_location.md]: why is this the endpoint and not the default endpoint?

File: articles/custom_storage_location.md [hunk 4]
```diff
@@ -457,6 +375,43 @@ projectDestination <- synRestPOST('/projectSettings', body=toJSON(projectDestina
 
 Navigate to your **Project/Folder -> Tools -> Change Storage Location**. In the resulting pop-up, select the **Google Cloud Storage Bucket** option and fill in the relevant information, where Bucket is the name of your external bucket, Base Key is the name of the folder in your bucket to upload to, and Banner is a short description such as who owns the storage location.
 
+### Adding Files in Your Google Cloud Bucket to Synapse
+
+If your bucket is set for read-write access, files can be added to the bucket using the standard Synapse interface (web or programmatic). 
+
+If the bucket is read-only or you already have content in the bucket, you will have to add representations of the files in Synapse programmatically. This is done using a `FileHandle`, which is a Synapse representation of the file.
```
Comment (kelshmo) [inline articles/custom_storage_location.md]:
```suggestion
If the bucket is read-only, or you already have content in the bucket, you will first create representations of the files in Synapse programmatically by creating a Synapse `File` to store a `dataFileHandleId`. 
```

Comment (kelshmo) [reply articles/custom_storage_location.md]: I see we do not define file handles anywhere in the documentation. As a follow-up, will create content describing this Synapse feature https://github.com/Sage-Bionetworks/synapseDocs/issues/743. This does not block merging this PR.

**Review by kelshmo — DISMISSED** [review]
Thanks Nick! I have one clarifying question and a suggestion to update how a file handle is described- small things and then this can be merged.

File: articles/custom_storage_location.md (hunk 2 above)
Comment (kelshmo) [reply articles/custom_storage_location.md]: resolving to merge PR

**Review by kelshmo — APPROVED** [review]

**Comment by nickgros** [comment]
@kelshmo Sorry to respond late to this PR.

re: Specifying `endpoint = 'https://file-prod.prod.sagebase.org/file/v1'`
>Why is this the endpoint and not the default endpoint?

The call is made to the file endpoint (https://repo-prod.prod.sagebase.org/file/v1/) rather than the default endpoint (https://repo-prod.prod.sagebase.org/repo/v1/).

At some point, presumably when the doc was first written, `synapseFileServiceEndpoint` existed in synapser, but it doesn't anymore, so that's why we hard-code the URL. This isn't an optimal solution, but I think that might be more of a Python/R client issue than a docs issue.

**Pull request merged**